code/src/App.js

```diff
@@ -1,9 +1,35 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit' 
+import { ToDoList } from 'reducers/ToDoList'
+
+import { Header } from 'components/Header'
+import { AddTask } from 'components/AddTask'
+import { TaskList } from 'components/TaskList'
+import { Counter } from 'components/Counter'
+import { ClearAll } from 'components/ClearAll'
+
+
+const reducer = combineReducers({
+  ToDoList: ToDoList.reducer
```

Comment:
Reducers should have lower-case first letters (like a normal function) - only components and classes should start with uppercase letters. So:

```js
const reducer = combineReducers({
  toDoList: toDoList.reducer
})
```

The reducer should also export the slice as `toDoList` and then any component which uses the reducer would also need to be updated.